portfolio/src/main/webapp/calendar.js

```diff
@@ -80,13 +80,21 @@ function createNewCalendarEventUserInput() {
     document.getElementById('new-event-name').value = EVENT_DEFAULT_NAME;
   }
 }
- 
+
+/** Shows the "Event added" header. */
+function showEventAddedHeader() {
+  const $eventListHeader = $('#event-added-header');
```

Comment:
You can merge these lines into,

$('#event-added-header').removeClass('d-none');

Reply:
Done.